samples/net/google_iot_device/src/config.h

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2019 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Declarations for application config options.
+ */
+
+#include <iotc_mqtt.h>
```

Comment:
> Is this include needed here?

A simple generic answer is that this header was here before. [And thus it would be stretching this patch too much to remove it.]

But yes, it's needed to declare stuff like `iotc_mqtt_qos_t` as referenced in this file.